### PR TITLE
Added a JumpUrl to reaction notifications

### PR DESCRIPTION
**Pull request by penev92**
Added a JumpUrl to reaction messages that link to the original message that got a reaction for easier tracking.

### CODE REVIEW

File: SteakBot.Core/EventHandlers/ReactionEventHandler.cs [hunk 1]
```diff
@@ -11,9 +11,17 @@ public async Task HandleReactionAddedAsync(Cacheable<IUserMessage, ulong> arg1,
         {
             var channel = arg3.Channel;
             var message = await channel.GetMessageAsync(arg3.MessageId);
+            var url = message.GetJumpUrl();
```
Comment (naps250) [inline SteakBot.Core/EventHandlers/ReactionEventHandler.cs]: Change name to something more precise (e.g. jumpUrl, messageUrl).

File: SteakBot.Core/EventHandlers/ReactionEventHandler.cs [hunk 2]
```diff
@@ -11,9 +11,17 @@ public async Task HandleReactionAddedAsync(Cacheable<IUserMessage, ulong> arg1,
...
 
-            await channel.SendMessageAsync($"{arg3.User.Value.Mention} reacted with {arg3.Emote} to a message by {message.Author.Username}"
-                                           + $" from {message.CreatedAt.LocalDateTime:HH:mm:ss} of {message.CreatedAt.LocalDateTime:dd.MM.yyyy}.");
+            var text = $"{arg3.User.Value.Mention} reacted with {arg3.Emote} to [a message]({url}) by {message.Author.Username}"
```
Comment (naps250) [inline SteakBot.Core/EventHandlers/ReactionEventHandler.cs]: Change var names to be more readable (text -> reactionMessage, embed -> embedMessage for example).

**Review by naps250 — DISMISSED** [review]
Review var names.

**Comment by penev92** [comment]
Fixed.

**Review by naps250 — APPROVED** [review]

**Pull request merged**